files/dhcp/vrf

```diff
@@ -0,0 +1,178 @@
+get_vrf_arg()
```

Comment:
can you provide description for this file? What is the purpose of this file?

Reply:
Will do.